.openapi-generator/generate.sh

```diff
@@ -1,6 +1,14 @@
+if [ -z "$1" ]
+  then
+    echo "Schema release version must be provided"
+    echo "eg: bash .openapi-generator/generate.sh v1.17.0"
+fi
+
+export VERSION=$1
+
 python .openapi-generator/pre_gen_script.py
-npx openapi-generator generate -i "https://www.ladybug.tools/honeybee-schema/simulation-parameter.json"  -t ".openapi-generator/templates/csharp" -g csharp -o . --skip-validate-spec -c .openapi-generator/config.json --type-mappings decimal=double 
-npx openapi-generator generate -i "https://www.ladybug.tools/honeybee-schema/model.json"  -t ".openapi-generator/templates/csharp" -g csharp -o . --skip-validate-spec -c .openapi-generator/config.json --type-mappings decimal=double
+npx openapi-generator generate -i "https://github.com/ladybug-tools/honeybee-schema/releases/download/$VERSION/simulation-parameter.json"  -t ".openapi-generator/templates/csharp" -g csharp -o . --skip-validate-spec -c .openapi-generator/config.json --type-mappings decimal=double 
+npx openapi-generator generate -i "https://github.com/ladybug-tools/honeybee-schema/releases/download/$VERSION/model.json"  -t ".openapi-generator/templates/csharp" -g csharp -o . --skip-validate-spec -c .openapi-generator/config.json --type-mappings decimal=double
```

Comment:
Hi @AntoineDao,
Can we access this json file from this link? Isn't GitHub hosting the content file at here: https://raw.githubusercontent.com?

Reply:
Yep, releases are different from git data. Gists and repo files are hosted on the raw.github domains. Releases can be accessed through a different API. Try running the script on your machine with v1.17.0 as an argument to check for yourself :)